### ISSUE TITLE
comment role author and UA requirements need a heavy rewrite or removal

### ISSUE THREAD

**Issue by cookiecrook** [w3c/aria#1885]
https://w3c.github.io/aria/#comment

Particularly this UA requirement:
> If the author has not explicitly declared [aria-level](https://w3c.github.io/aria/#aria-level), [aria-posinset](https://w3c.github.io/aria/#aria-posinset), or [aria-setsize](https://w3c.github.io/aria/#aria-setsize) for a comment element, user agents MUST automatically compute the missing values and expose them to assistive technologies.

This would need some algorithm defined that would almost certainly be BOTH faulty AND too rigid. I propose removing it entirely.

But the author rules about level and details are passive, vague, and contradictory too. I recommend those be pulled from the spec, and possibly rewritten for the APG.
